docs/design.md

```diff
@@ -4,7 +4,8 @@ by Spencer Kimball from early 2014.
 
 # Overview
 
-Cockroach is a distributed key:value datastore which supports **ACID
+Cockroach is a distributed key:value datastore (SQL and structured
+data layers of cockroach have yet to be defined) which supports **ACID
```

Comment:
I have a slight preference for wrapping at 80 chars (or some other width, but I wouldn't want to have a different wrap width for markdown files than for comments in the code), although I don't have any reason for it other than tradition and wouldn't object to unlimited-length lines in prose docs.